homework/style.css

```diff
@@ -1,3 +1,48 @@
 .alert-error {
```

Comment:
Your UI layout looks a bit sloppy. The repository values are not ligned out vertically, the colons (`:`) need proper spacing and long repository values wrap to the extreme left margin, especially visible on a mobile device. Maybe you can solve a couple of layout issues by using an HTML table for each repo?

![image](https://user-images.githubusercontent.com/2788771/65320873-221d2b80-dba3-11e9-8ee9-2e5f3f339e9e.png)

On wide-screen monitor your UI looks too streched out. Perhaps set a max width?

![image](https://user-images.githubusercontent.com/2788771/65320963-52fd6080-dba3-11e9-8899-46a7a74eb14b.png)

Reply:
About the stretch issue, I gave percentage value to widths and margins.